commands/roles.go

```diff
@@ -0,0 +1,68 @@
+package commands
+
+import (
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+type Role struct {
+	names []string
+	desc  string
+	role  string
+}
+
+func NewRole(role string) *Role {
+	return &Role{
+		names: []string{strings.ToLower(role)},
+		desc:  "Gives user the " + role + " role.",
+		role:  role,
+	}
+}
+
+func (r *Role) Names() []string {
+	return r.names
+}
+
+func (r *Role) Desc() string {
+	return r.desc
+}
+
+func (r *Role) Roles() []string {
+	return nil
+}
+
+func (r *Role) Chans() []string {
+	return nil
+}
+
+func (r *Role) MsgHandle(ses *discordgo.Session, msg *discordgo.Message, args []string) (*CommandSend, error) {
+	member, _ := ses.GuildMember(msg.GuildID, msg.Author.ID)
+
+	// Find the role ID for this role name
+	var roleID string
+	guildRoles, _ := ses.GuildRoles(msg.GuildID)
```

Comment:
```suggestion
	guildRoles, err := ses.GuildRoles(msg.GuildID)
	if err != nil {
	    return _, err
	}
```

Reply:
Should always pass up errors when possible until they are handled (in this case, the caller to `MsgHandle` in main will handle the message by logging it). There is no good reason to ignore the error.